### ISSUE TITLE
Option for specifying custom asset folder?

### ISSUE THREAD

**Issue by trnc-ck** [Cowboy-coder/bundle-up#22]
hi,

is there a way to specify a custom folder name fpr the generated assets? instead of the given

> /generated/bundle/037e6e5_global.css

i want to use

> /assets/037e6e5_global.css

any advice? thanks!

**Comment by Cowboy-coder** [comment]
No, there is nothing in the api currently that exposes the functionality to change that path for generated files. Why do you need it?

The only thing that you can change is `staticRoot`. Like `__dirname + "/assets/"` instead of `__dirname + "/public/"` for example.

**Comment by zthomas** [comment]
+1 This is actually pretty crucial, since a lot of css files reference images and other assets using a relative path, using /generate/bundle breaks all my css. It's really preventing me from using this in production.
